src/main/java/WorkWithTheConsole.java

```diff
@@ -4,15 +4,15 @@
 
 public class WorkWithTheConsole {
     public static void messageForUsers() {
-        System.out.println("Приветствую! Это калькулятор, который может высчитывать несколько действий сразу!" +
-                " Введи какое нибудь выражение и нажми \"Enter\" ");
+        System.out.println("Приветствую! Это калькулятор, который может производить вычисления с положительными дробными числами!" +
+                " Введи своё выражение и нажми \"Enter\" ");
     }
 
     public static String getExpression() {
         return new Scanner(System.in).nextLine();
     }
 
     public static void outputToTheConsoleOfTheResponse() {
-        System.out.println("Ответ: " + MathematicalCalculations.Calculations());
+        System.out.println("Ответ: " + MathematicalCalculations.Calculations(ExpressionParser.parseExpressionInReversePolandNotation(getExpression())));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.